SU-SWS/decanter
# D8 core 724 button hover

# READY FOR REVIEW

# Summary

Hover and focus should rely on something other that a color change. I put in an underline to indicate the change
https://stanfordits.atlassian.net/browse/D8CORE-724

# Needed By (Date)
- When does this need to be merged by?
Soon.

# Urgency
- How critical is this PR?
Would like it for DecanterPalooza

# Steps to Test

Check out this branch
Verify the button hover, active, and focus work as expected

# Affected Projects or Products

# Associated Issues and/or People
- JIRA ticket: https://stanfordits.atlassian.net/browse/D8CORE-724

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
